### DISCUSSION TITLE
Saving the results (detected class) of trained yolov5 model.

### DISCUSSION THREAD

**Discussion by rabiyaabbasi** [ultralytics/yolov5#7787] [Q&A]
Hi, Is there any way to save the results obtained from running the command "detection.pandas().xyxy[0]". For reference, I am using the below code. 
{

model = torch.hub.load('ultralytics/yolov5', 'custom', path='yolov5/runs/train/exp4/weights/best.pt', force_reload=True) 
img = cv2.imread('ba7.jpg')[:, :, ::-1] 
deetection = model(img, size=416)  
detection.pandas().xyxy[0]

}

The command "detection.pandas().xyxy[0]" just displays the results but I want to save those results in csv or txt file. My results after running this command are shown below. 

![image](https://user-images.githubusercontent.com/54917838/168119856-8cb1ef74-fef5-4ea5-938b-bcf9cb843ff4.png)

**Comment by glenn-jocher** [comment]
@rabiyaabbasi sure, you can do anything you want with the results! To convert to CSV see pandas docs: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html
